packages/manager/src/features/Databases/DatabaseDetail/DatabaseResize/DatabaseResizeCurrentConfiguration.tsx

```diff
@@ -139,7 +139,9 @@ export const DatabaseResizeCurrentConfiguration = ({ database }: Props) => {
               </StyledSummaryTextTypography>
               <StyledSummaryTextTypography>
                 <span style={{ fontFamily: theme.font.bold }}>Used</span>{' '}
-                {database.used_disk_size_gb} GB
+                {database.used_disk_size_gb
```

Comment:
We may want to explicitly check `!== null` so that if the API returns 0, `0 GB` will be shown rather than `N/A`

```suggestion
                {database.used_disk_size_gb !== null
```

Reply:
@bnussman-akamai Thanks, that makes sense given that it's an integer. I've pushed up this change!